### ISSUE TITLE
@after method to be provided with the response

### ISSUE THREAD

**Issue by tyllo** [vlio20/utils-decorators#117]
in lines  https://github.com/vlio20/utils-decorators/blob/cb2ebded3fbe76deeb18590b277898b20626061f/src/after/afterify.ts#L19 https://github.com/vlio20/utils-decorators/blob/cb2ebded3fbe76deeb18590b277898b20626061f/src/after/afterify.ts#L25 need return response for origin method

**Comment by vlio20** [comment]
Hi @tyllo, not sure I understand. The after decorator is called with both the original arguments and the original method response.

Maybe you can provide a code example with your expectation?

**Comment by tyllo** [comment]
Sorry for my bad englesh
```javascript
import { afterify } from 'utils-decorators';

function foo = (value) => {
  console.log('original method called');
  return value;
}

const fooWithAfter = afterify(foo, {
  func: (respData: AfterParams<number>) => {
      console.log(respData.args, respData.response);
  };
});

const isEqual = foo(42) === fooWithAfter(42)
console.log('Expect result is 42', isEqual)
``

**Comment by vlio20** [comment]
I understand your requirement. Seems like this is relevant only for the afterify function and not for the @after decorator.

@tyllo would you like to submit a PR?

**Comment by tyllo** [comment]
This issue effected and @after decorator

**Comment by vlio20** [comment]
You are right. In both cases the value should be returned. Good point.

Would you like to submit a PR?

**Comment by vlio20** [comment]
Thanks, @tyllo. The fix is available in version 1.10.2
